content/websockets/gateways.md

```diff
@@ -54,7 +54,22 @@ handleEvent(data) {
 
 > info **Hint** `@SubscribeMessage()` and `@MessageBody()` decorators are imported from `@nestjs/websockets` package.
 
-The `handleEvent()` function takes two arguments. First one is a platform-specific [socket instance](https://socket.io/docs/server-api/#socket) and the second one is the data received from the client.
+In case you don't want to use decorators, the following code will be valid as well:
```

Comment:
If you would prefer not to use decorators, the following code is functionally equivalent: